modules/packages/Makefile

```diff
@@ -13,18 +14,23 @@ packages/reinstall: packages/delete packages/install
 ## Install packages 
 packages/install:
 	@if [ ! -d $(PACKAGES_PATH) ]; then \
-	  echo "Installing packages $(PACKAGES_VERSION)..."; \
+	  echo "* Installing packages $(PACKAGES_VERSION)..."; \
 	  rm -rf $(PACKAGES_PATH); \
 	  $(GIT) clone -c advice.detachedHead=false --depth=1 -b $(PACKAGES_VERSION) https://github.com/cloudposse/packages.git $(PACKAGES_PATH); \
 	  rm -rf $(PACKAGES_PATH)/.git; \
 	fi 
 
 ## Install package (e.g. helm, helmfile, kubectl)
-packages/install/%: packages/install
-	@if [ ! -x $(INSTALL_PATH)/$(subst packages/install/,,$@) ]; then \
-		$(MAKE) -C $(PACKAGES_PATH)/install $(subst packages/install/,,$@); \
+packages/install/%:
+	@binary="$*"; \
+	if [[ -x "$(INSTALL_PATH)/$$binary" ]]; then \
+  		echo "* Package $$binary already installed"; \
+	elif [[ "$(PACKAGES_PREFER_HOST)" == "true" ]] && installed=$$(command -v $* 2>/dev/null); then \
+			echo Using "$*" from "$$installed" ; \
+			cp -p "$$installed" "$*" ; \
```

Comment:
`ln`?

Reply:
I originally wrote `ln` but then realized that the `vendor` directory will probably be on the host and the `bin` directory could be in the Docker image, so `ln` could break things for one or the other.

Reply:
Actually, we can do without `cp` or `ln`, since the executable is already on the `PATH`